cmd/gotk/get_kustomization.go

```diff
@@ -50,37 +53,44 @@ func getKsCmdRun(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
+	table := tablewriter.NewWriter(os.Stdout)
+	table.SetHeader([]string{"Name", "Revision", "Ready Status", "Message", "Namespace"})
+	table.SetRowLine(true)
+
 	var list kustomizev1.KustomizationList
 	err = kubeClient.List(ctx, &list, client.InNamespace(namespace))
 	if err != nil {
 		return err
 	}
 
 	if len(list.Items) == 0 {
-		logger.Failuref("no kustomizations found in %s namespace", namespace)
+		table.Append([]string{"None", "None", "Unknown", "No kustomizations found", namespace})
+		table.Render()
```

Comment:
If there are no kustomizations found, this should not print a table but simply log a string to `stderr` (as it did before the change).